pkg/kubelet/apis/config/fuzzer/fuzzer.go

```diff
@@ -119,6 +119,7 @@ func Funcs(codecs runtimeserializer.CodecFactory) []interface{} {
 			obj.EnableSystemLogHandler = true
 			obj.MemoryThrottlingFactor = utilpointer.Float64(rand.Float64())
 			obj.LocalStorageCapacityIsolation = true
+			obj.PullImageSecretRecheckDuration = metav1.Duration{Duration: 24 * time.Hour}
```

Comment:
This 24h duration is used in a few spots. Can you create a constant for the RecheckDuration?

Reply:
Add a `DefaultRecheckDuration`.